src/App.jsx

```diff
@@ -14,45 +16,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const getPreparedGoods = (goods, sortField, isReversed) => {
+  const visibleGoods = [...goods];
+
+  if (sortField) {
+    visibleGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SORT_FIELD_LENGTH:
+          return good1.length - good2.length;
+
+        case SORT_FIELD_NAME:
+          return good1.localeCompare(good2);
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (isReversed) {
+    visibleGoods.reverse();
+  }
+
+  return visibleGoods;
+};
+
+const SORT_FIELD_NAME = 'name';
+const SORT_FIELD_LENGTH = 'length';
```

Comment:
better to make object with possible sort fields and use them.


```suggestion
const SORT__FIELD = {
  NONE: 'none',
  NAME: 'name',
  LENGTH: 'length',
}
```